RELEASE-LEADS.md

```diff
@@ -238,61 +154,92 @@ green light.
 
 ## Day of the release
 
-### Cut `release-x.y` branches of `serving` and `eventing`
+Follow the [cutting release branches](#cutting_release_branches) instructions
+for each repo. Wait for release automation to kick in (runs on a 2 hour
+interval). Once the release automation passed, it will create a release tag in
+the repositorie. Enhance the respective tags with the collected release-notes
+using the Github UI.
 
-Create a `release-x.y` branch from master in both repositories. Wait for release
-automation to kick in (runs on a 2 hour interval). Once the release automation
-passed, it will create a release tag in both repositories. Enhance the
-respective tags with the collected release-notes using the Github UI.
+In general the release dependency order is something like the following (as of
```

Comment:
I think a future release lead might be quite worried by "In general" and "something like" here. Will something tell us if we get the order wrong? Can we give more of a hint on how to verify the order, if it may be incorrect?

Reply:
buoy check will fail and tell you if the repo is not ready.

Reply:
maybe we can add something to the doc to reflect that?

Reply:
I did :D